images/win/scripts/Tests/VisualStudio.Tests.ps1

```diff
@@ -15,7 +15,19 @@ Describe "Visual Studio" {
         $expectedComponents = Get-ToolsetContent | Select-Object -ExpandProperty visualStudio | Select-Object -ExpandProperty workloads
         $testCases = $expectedComponents | ForEach-Object { @{ComponentName = $_} }
         BeforeAll {
-            $installedComponents = Get-VisualStudioComponents | Select-Object -ExpandProperty Package
+            $installedComponents = Get-VisualStudioComponents -ProductType "VisualStudio" | Select-Object -ExpandProperty Package
+        }
+
+        It "<ComponentName>" -TestCases $testCases {
+            $installedComponents | Should -Contain $ComponentName
+        }
+    }
+
+    Context "Visual Studio Build Tools components" {
+        $expectedComponents = Get-ToolsetContent | Select-Object -ExpandProperty visualStudio | Select-Object -ExpandProperty buildtools_workloads
```

Comment:
nitpick: (Get-ToolsetContent).visualStudio.buildtools_workloads

Reply:
ye, it looks much less sophisticated